index.js

```diff
@@ -3,13 +3,17 @@
 	Author Tobias Koppers @sokra
 */
 var path = require("path");
+var loaderUtils = require("loader-utils");
```

Comment:
We forget add `loader-utils` to `package.json` :smile:

Reply:
Oh you're right ! I added it.